Link.js

```diff
@@ -0,0 +1,15 @@
+class Link {
+    href
+    text
+    file
+
+    constructor(href, text, file) {
+        this.href = href;
+        this.text = text;
+        this.file = file;
```

Comment:
Se entiende la complejidad el keyword this y que significa estar en el scope incorrecto

Reply:
.bind(this)

Reply:
y que relación tiene con ()=>{}